fixed_bed_doe.ipynb

```diff
@@ -0,0 +1,1592 @@
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      " 330r-9.1431443e+00 5.70e-01 2.17e+02  -9.0 3.99e+01    -  1.00e+00 6.45e-02h  1\n",
+      " 331r-9.2069822e+00 6.15e-01 1.82e+02  -9.0 3.91e+01    -  1.00e+00 1.47e-01h  1\n",
+      " 332r-9.2297030e+00 5.95e-01 1.64e+02  -9.0 3.67e+01    -  1.00e+00 5.89e-02h  1\n",
+      " 333r-9.2779197e+00 5.93e-01 1.39e+02  -9.0 3.51e+01    -  1.00e+00 1.33e-01h  1\n",
+      " 334r-9.3003831e+00 5.68e-01 1.38e+02  -9.0 3.25e+01    -  1.00e+00 7.15e-02h  1\n",
+      " 335r-9.3686532e+00 6.03e-01 1.05e+02  -9.0 3.11e+01    -  1.00e+00 2.33e-01h  1\n",
+      " 336r-9.3883004e+00 5.66e-01 8.95e+01  -9.0 2.55e+01    -  1.00e+00 8.47e-02h  1\n",
+      " 337r-9.3901454e+00 5.61e-01 1.11e+02  -9.0 2.38e+01    -  1.00e+00 9.02e-03h  1\n",
+      " 338r-9.4399702e+00 5.23e-01 9.14e+01  -9.0 2.37e+01    -  1.00e+00 2.50e-01h  1\n",
+      " 339r-9.4916631e+00 4.58e-01 5.92e+01  -9.0 1.90e+01    -  1.00e+00 3.37e-01h  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      " 340r-9.5673570e+00 3.67e-01 1.58e+01  -9.0 1.36e+01    -  1.00e+00 7.22e-01h  1\n",
+      " 341r-9.5966767e+00 6.01e-02 1.03e+00  -9.0 4.31e+00    -  1.00e+00 9.36e-01h  1\n",
+      " 342r-9.5968198e+00 5.62e-02 4.98e+00  -9.0 3.72e-01    -  1.00e+00 6.50e-02h  1\n",
+      " 343r-9.5990702e+00 4.25e-04 3.94e-06  -9.0 3.57e-01    -  1.00e+00 1.00e+00h  1\n",
+      " 344r-9.5990941e+00 3.06e-04 1.29e-08  -9.0 3.51e-03    -  1.00e+00 1.00e+00h  1\n",
+      "\n",
+      "Number of Iterations....: 344\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:  -9.5990963042971167e+00   -9.5990963042971167e+00\n",
+      "Dual infeasibility......:   2.5253574455622485e-01    2.5253574455622485e-01\n",
+      "Constraint violation....:   3.0573204779631136e-04    3.0573204779631136e-04\n",
+      "Complementarity.........:   9.0909090909091044e-10    9.0909090909091044e-10\n",
+      "Overall NLP error.......:   3.0573204779631136e-04    2.5253574455622485e-01\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 348\n",
+      "Number of objective gradient evaluations             = 6\n",
+      "Number of equality constraint evaluations            = 348\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 347\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 345\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =    976.407\n",
+      "Total CPU secs in NLP function evaluations           =    119.932\n",
+      "\n",
+      "EXIT: Converged to a point of local infeasibility. Problem may be infeasible.\n"
```

Comment:
We should debug this. Why does Ipopt always go to restoration failure? Which equations are infeasible? You can use degeneracy hunter.